from core.exceptions.vault_k8s import VaultException